Source/server.js

```diff
@@ -14,6 +14,8 @@ var morgan       = require('morgan');
 var cookieParser = require('cookie-parser');
 var bodyParser   = require('body-parser');
 var session      = require('express-session');
+var mongoose = require("mongoose");
+mongoose.connect("mongodb://localhost:27017/");
```

Comment:
this is already on line 22

Reply:
Not sure why that duplicated over. Removed!